introduction_to_amazon_algorithms/jumpstart_image_classification/Amazon_JumpStart_Image_Classification.ipynb

```diff
@@ -0,0 +1,661 @@
+{
```

Comment:
<p>@julkroll do we need have these files in sagemaker-sample-files to unify our data sources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>These files are currently downloaded from public s3 buckets of JumpStart. </p><p> </p><p>I understand that it would be best to do so. But if it is not a critical thing, can we do it in the next revision?</p>